dev/x86_64/src/arith_native_x86_64.h

```diff
@@ -11,9 +11,9 @@
 #include <stdint.h>
 #include "consts.h"
 
-#define REJ_UNIFORM_AVX_NBLOCKS 3 /* See MLKEM_GEN_MATRIX_NBLOCKS */
-#define REJ_UNIFORM_AVX_BUFLEN \
-  (3 * 168) /* REJ_UNIFORM_AVX_BUFLEN * SHAKE128_RATE */
+#define MLK_AVX2_REJ_UNIFORM_NBLOCKS 3 /* See MLKEM_GEN_MATRIX_NBLOCKS */
+#define MLK_AVX2_REJ_UNIFORM_BUFLEN \
+  (3 * 168) /* REJ_UNIFORM_BUFLEN * SHAKE128_RATE */
```

Comment:
I don't quite understand this comment. should be nblocks I suppose?

Reply:
Done